charts/seed-bootstrap/charts/elastic-kibana-curator/templates/kibana/kibana-index-registration-config.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kibana-index-registration
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: kibana
+    role: logging
+data:
+  register: |-
+    #/bin/sh
+
+    until curl -sS http://127.0.0.1:{{ .Values.kibanaPort }}/ > /dev/null; do
+      echo Waiting for Kibana...;
+      sleep 2;
+    done;
+
+    function register {
+      curl -sS -X$1 -H "Content-Type: application/json" -H "kbn-xsrf: anything" \
+      'http://127.0.01:{{ .Values.kibanaPort }}/api/saved_objects/index-pattern/logstash-*' \
+      -d'{"attributes":{"title":"logstash-*","timeFieldName":"@timestamp"}}'
+      echo -e "\n"
+    }
+
+    # curl doesn't care about HTTP status codes.
```

Comment:
What is meant by that comment?

Reply:
To curl `HTTP 200 OK` and `HTTP 404 NotFound` are the same - they don't cause curl to exit with an error.

Reply:
```bash
$ code=$(curl --silent --output /dev/stderr --write-out "%{http_code}" localhost:8001/apis/foo/bar)
$ echo $code
404
```